Resources/Prototypes/Entities/Objects/Misc/implanters.yml

```diff
@@ -251,3 +251,11 @@
   components:
     - type: Implanter
       implant: MindShieldImplant
+
+- type: entity
+  id: lifesaverImplanter
+  name: Life saver implanter
+  parent: BaseImplantOnlyImplanter
+  components:
+    - type: Implanter
+      implant: LifeSaverImplant
```

Comment:
Don't add dummy entities, remove please.

Reply:
reverted changers to implanter and subdermalimplant